examples/diff.c

```diff
@@ -335,3 +363,27 @@ static void diff_print_stats(git_diff *diff, struct opts *o)
 	git_buf_free(&b);
 	git_diff_stats_free(stats);
 }
+
+int extract_break_rewrites(int *m, int *n, const char *arg)
+{
```

Comment:
- Comments would be nice on the 2 main `if`s. I can't tell quickly enough was happens, and *I'm the one that proposed it* (and I remember adding some because "clarity").
- Function returns either `1` or `-1`, both of which are not 0, and thus always "false". In C, `0` means success, everything else is "error", because if someone does the obvious `if(!function) exit(-1);` they'll exit with wonder.
- m_str/n_str are unused.